framework/grid/ActionColumn.php

```diff
@@ -153,33 +153,53 @@ protected function initDefaultButtons()
     /**
      * Initializes the default button rendering callback for single button.
      * @param string $name Button name as it's written in template
-     * @param string $iconName The part of Bootstrap glyphicon class that makes it unique
      * @param array $additionalOptions Array of additional options
      * @since 2.0.11
      */
-    protected function initDefaultButton($name, $iconName, $additionalOptions = [])
+    protected function initDefaultButton($name, $additionalOptions = [])
```

Comment:
That could be a problem for classes extending `ActionColumn`. How about leaving $iconName but marking it as `@deprecated` instead?

Reply:
yes, you are right. I make it so.